setup.py

```diff
@@ -40,11 +40,11 @@ def get_static_files(path):
 install_requires = [
     'tornado>=4.0,<5',
     'python-daemon<3.0',
+    # So that we can build documentation for luigi.db_task_history and luigi.contrib.sqla
```

Comment:
Thank you for pinning `sqlalchemy` to 1.4.32. In working on upgrading our fork to 2.8.13, one of my biggest challenges was unpinned dependencies that have since dropped support for Python 2.

Reply:
I feel like there is a case for leaving `setup.py` as is, but instead install `sqlalchemy` in a virtual environment and launch luigid in that virtual environment. I think that is what we already do.

If you want to leave this as is, I would at least update the comment to something like "sqlalchemy is a requirement for writing task records to the Luigi history server. We could install it in the luigi venvironment, be we just made it a dependency here instead.”

Reply:
I think it would be cool to be able to launch from setup, using our fork on the scheduler. And have it then pinned there, so I think I will lean to making the base setup something that works OOB. :P